pkg/asset/ignition/bootstrap/bootstrap_test.go

```diff
@@ -0,0 +1,67 @@
+package bootstrap
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	igntypes "github.com/coreos/ignition/config/v2_2/types"
+	"github.com/golang/mock/gomock"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/mock"
+)
+
+func TestBootstrapLoad(t *testing.T) {
+	tests := []struct {
+		name       string
+		config     *igntypes.Config
```

Comment:
Might have made sense to use the marshaled json here to avoid the marshaling and and to allow testing invalid json

Reply:
ok, I'll change that and test the Marshal error path too. Thanks for the review!